tests/src/CLRTest.Execute.Batch.targets

```diff
@@ -18,6 +18,33 @@ WARNING:   When setting properties based on their current state (for example:
 -->
 <Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
 
+  <ItemGroup>
+    <CLRTestEnvironmentVariable Condition="'$(CrossGen)' == 'true'" Include="set complus_zaprequire=1" />
+    <CLRTestEnvironmentVariable Condition="'$(CrossGen)' == 'true'" Include="set complus_zaprequireexcludelist=corerun" />
```

Comment:
I'm surprised that it's necessary to add corerun to zaprequireexcludelist, since it's an unmanaged app.